### PR TITLE
cram: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**
Required by https://github.com/conda-forge/staged-recipes/pull/3110 for running tests.

### CODE REVIEW

File: recipes/cram/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - cram
+
+  commands:
+    - cram -h
+
+about:
+  home: https://bitheap.org/cram/
+  license: GPL-2.0
+  summary: 'A simple testing framework for command line applications'
+  license_family: GPL2
```
Comment (isuruf) [inline recipes/cram/meta.yaml]: Is the license not in the source tarball? If not, please raise an issue upstream.
See http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

Comment (nehaljwani) [inline recipes/cram/meta.yaml]: Can I use this file? https://github.com/brodie/cram/blob/master/COPYING.txt

Comment (isuruf) [inline recipes/cram/meta.yaml]: Yes.

Comment (nehaljwani) [inline recipes/cram/meta.yaml]: It's available in the source tarball and I have updated the recipe.

File: recipes/cram/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{% set name = "cram" %}
+{% set version = "0.7" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "md5" %}
```
Comment (isuruf) [inline recipes/cram/meta.yaml]: sha256 is preferred. https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L7

Comment (nehaljwani) [inline recipes/cram/meta.yaml]: Updated.

**Pull request merged**